datafusion/core/tests/physical_optimizer/enforce_distribution.rs

```diff
@@ -647,12 +630,18 @@ fn multi_hash_joins() -> Result<()> {
                         "      DataSourceExec: file_groups={1 group: [[x]]}, projection=[a, b, c, d, e], file_type=parquet",
                     ],
                 };
-                assert_optimized!(
-                    expected,
+
+                let test_config = TestConfig::default();
+                test_config.run(
+                    &expected,
                     top_join.clone(),
-                    &TestConfig::new(DoFirst::Distribution)
-                );
-                assert_optimized!(expected, top_join, &TestConfig::new(DoFirst::Sorting));
+                    DISTRIB_DISTRIB_SORT.into(), // same if distribution enforced before sort.
```

Comment:
a minor nitpick is I don't think these extra comments add much additional value. It could be

```suggestion
                    DISTRIB_DISTRIB_SORT.into(), 
```